.github/workflows/golangci-lint.yml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2023 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
strictly speaking it's a copy taken from some example, not our original work

Reply:
should I remove it? it's always added automatically when running `license` as part of `before_commit` make

Reply:
probably yes, it would be more fair :)